Reproducible Comparing jobs didn't show as part of dev.functional jenkins job

Reproducible Comparing jobs were enabled as part of dev.functional, which didn't run in https://ci.adoptium.net/job/Test_openjdk21_hs_dev.functional_x86-64_linux/2/

Issues known:

- by default VENDOR_TEST_REPOS parameter of  [dev.functional job](https://ci.adoptium.net/job/Test_openjdk21_hs_dev.functional_x86-64_linux ) parameter VENDOR_TEST_REPOS is not set.
- dev.functional can't guarantee running on agent with 'docker installed'. Plus we can't force dev.functional running with docker installed only as ssl-tests, installler wintps tests don't or couldn't require docker installed.
- `jq` installed, but job complains `jq: error: Could not open file : No such file or directory` `13:23:46   jq                    x86_64      1.6-2.el7                 epel         167 k` - this one I met before can't remember the reason
- with dev.functional as target job failed with `[make[1]: *** [settings.mk:356: dev.functional-..] Terminated](https://ci.adoptium.net/job/Test_openjdk21_hs_dev.functional_x86-64_linux/6/console)` -- This error feels like intermittent, rerun on same machine and didn't get the error https://ci.adoptium.net/view/Test_functional/job/Test_openjdk21_hs_dev.functional_x86-64_linux/6/ https://ci.adoptium.net/view/Test_functional/job/Test_openjdk21_hs_dev.functional_x86-64_linux/8/
-

The `[make[1]: *** [settings.mk:356: dev.functional-..] Terminated](https://ci.adoptium.net/job/Test_openjdk21_hs_dev.functional_x86-64_linux/6/console)` issue mentioned above was related to a dockerhost node being labelled with ci.role.test.  dockerhost nodes should not get that label, as they host several static test nodes, and if tests are run directly on dockerhost, then tests on the static nodes are randomly Terminated.  (see discussion in [Slack thread](https://adoptium.slack.com/archives/C09NW3L2J/p1704471126923519))

--- a/test/functional/reproducibleCompare/playlist.xml
+++ b/test/functional/reproducibleCompare/playlist.xml
@@ -25,7 +25,7 @@
 			<level>dev</level>
 		</levels>
 		<groups>
-			<group>functional</group>
+			<group>system</group>
 		</groups>
         <vendors>
             <vendor>eclipse</vendor>

